packages/common/src/schema.ts

```diff
@@ -56,6 +56,7 @@ export const Dict = z.object({
   'schemas.text.max': z.string(),
   'schemas.text.fit': z.string(),
   'schemas.text.dynamicFontSize': z.string(),
+  'schemas.text.format': z.string(),
```

Comment:
do we still need this?

Reply:
Yes. It is for the [schema title](https://github.com/pdfme/pdfme/pull/494/files#diff-84fadaad049676d98756eacc6206f5f695bca78fe837522dd9b68bca10cb73a6R71-R72).

Reply:
ah sorry, yes this is translations not schema definition 👍